File: Ad-Hoc/2437/2437.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        string[] input = Console.ReadLine().Split(' ');

        int x1 = int.Parse(input[0]);
        int y1 = int.Parse(input[1]);
        int x2 = int.Parse(input[2]);
        int y2 = int.Parse(input[3]);

        int distanceManhattan = Math.Abs(x1 - x2) + Math.Abs(y1 - y2);

        Console.WriteLine(distanceManhattan);
    }
}

File: Ad-Hoc/2437/2437.ts
import * as fs from "fs"
const input = fs.readFileSync("/dev/stdin", "utf8")
const [x1, y1, x2, y2]: number[] = input.split(' ').map(Number)

const distance_manhattan: number = Math.abs(x1 - x2) + Math.abs(y1 - y2)

console.log(distance_manhattan)